### PR TITLE
Next web clone

### PR DESCRIPTION
**Pull request by Lameck1**
### What does this do? 
* Replicating "The Next Web" website putting emphasis on how it behaves differently depending on the size of the screen where the website is rendered.

### Description of Task to be completed? 
* Usage of Flex and Grid for positioning of the page content
* Usage of media queries to ensure page responsiveness on variable screens

### How should this be manually tested? 
* Clone this repo https://github.com/yocosaka/building-with-responsive-design
* Navigate to project directory.
* If you have a live server in you vs-code, just run it.
* Otherwise just open the index page with your browser.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 yarn-error.log
 yarn.lock
+.vscode/
```
Comment (arslanbisharat) [inline .gitignore]: I can see you added .vscode in gitignore but it is still in your repo. Please resolve this

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

**Project Updates**

- [ ] At medium and small screens this logo is not aligned correctly as present on the original page. Please work on it. 
chrome-capture (12)


**Repo Updates** 

- [x] You can add some extra files to gitignore. By this, your repo will look cleaner. 

- [x] Please add more description in your project about section. This can help to understand what your project is about. 
chrome-capture (11)

- [x] I can see you added .vscode in gitignore but it is still in your repo. Please resolve this.



**Updates in Readme**

- [x] You can remove **PROJECT NAME** from your readme.
chrome-capture (9)

- [x] Your readme is great. But you can find a better template. [Here is an example.](https://github.com/othneildrew/Best-README-Template) 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by Lameck1** [comment]
Hello TSE,

We have done work on the recommendations made, though we can't seem to notice an issue with the rendering of the logo as in your case. Both my partner and I, have tested the rendering of the page on our local machines and the logo renders perfectly. Maybe you should advise us on the browser that you are using.

Thanks for your review, we are waiting for further feedback.

**Comment by arslanbisharat** [comment]
> Hello TSE,
> 
> We have done work on the recommendations made, though we can't seem to notice an issue with the rendering of the logo as in your case. Both my partner and I, have tested the rendering of the page on our local machines and the logo renders perfectly. Maybe you should advise us on the browser that you are using.
> 
> Thanks for your review, we are waiting for further feedback.

You don't need to add these comments.

**Review by arslanbisharat — APPROVED** [review]
## Congratslutons Approved. 🎉
----------------------------------------------------------------------------------------------------------
Hello Developer 👋
You are on 🔥
Thank you so much for making those changes and I am very happy to approve the project. You have done a great job. 👏
Please keep working hard and move forward with your curriculum.
Looking forward to seeing your next project. 😄
---------------------------------------------------------------------------------------------------------------
**To highlight**
- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**